packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -442,3 +579,61 @@ class _ConstantValueListenable<T> implements ValueListenable<T> {
   @override
   final T value;
 }
+
+/// Immutable compound values that describe the effective ticker behavior
+/// for a subtree.
+///
+/// Instances of this class are produced by [TickerMode.valuesOf] and
+/// [TickerMode.getValuesNotifier] and reflect the values that apply at a given
+/// location in the widget tree after taking ancestor [TickerMode] widgets into
+/// account.
+///
+/// Semantics of the fields:
+/// - [enabled]: A ticker is considered enabled only if all ancestor
+///   [TickerMode.enabled] values and the local [TickerMode.enabled] are true
+///   (logical AND). When false, tickers are muted (time still elapses but
+///   callbacks are not invoked).
+/// - [forceFrames]: When true, tickers in the subtree request frames using
+///   [SchedulerBinding.scheduleForcedFrame] while active. This value is
+///   combined across ancestors using logical OR, so any ancestor requesting
+///   forced frames enables it for the subtree.
+///
+/// For most widgets, reading these values is unnecessary; mixins such as
+/// [SingleTickerProviderStateMixin] and [TickerProviderStateMixin] apply them
+/// automatically to the [Ticker]s they vend. Use this class when you need to
+/// observe or react to ticker policy explicitly.
+class TickerModeData {
```

Comment:
```suggestion
@immutable
class TickerModeData {
```

Reply:
Indeed, so obvious I should have seen that, my bad!